Lib/email/_header_value_parser.py

```diff
@@ -341,9 +341,7 @@ def _fold(self, folded):
             # avoid infinite recursion.
             ws = part.pop_leading_fws()
             if ws is not None:
-                # Peel off the leading whitespace and make it sticky, to
-                # avoid infinite recursion.
-                folded.stickyspace = str(part.pop(0))
+                folded.stickyspace = ws
```

Comment:
It looks like this was a case of me "fixing" the code, and missing one spot (for which there obviously wasn't a test).  To make this code match other other instances of this logic, you should put an str call around ws.